spec/Overview.html

```diff
@@ -612,24 +612,26 @@ <h2>Terminology</h2>
           {{BufferSource}} are defined in [[WEBIDL]].
         </p>
         <p>
-          An <dfn id="dfn-octet-string">octet string</dfn> is an ordered sequence of zero or more
-          integers, each in the range 0 to 255 inclusive.
+          The term [= byte sequence =] is defined in [[Infra]].
```

Comment:
Typically you have a line like this:

> This specification depends on the Infra Standard. [INFRA]

Reply:
Yeah, I was looking at that for https://github.com/w3c/webcrypto/issues/268, but there's still a few other inconsistencies with the language there, so I'll add that later :blush: